bench/bench-bytestring.cabal

```diff
@@ -63,6 +82,10 @@ executable bench-bytestring-builder
                     -fdicts-cheap
                     -fspec-constr-count=6
 
+  -- blaze-builder-0.3.* requires a GHC < 8.4
+  if impl(ghc >= 8.4)
+    buildable: false
```

Comment:
I've simply disabled building `bench-bytestring-builder` with GHC >= 8.4 for now.

Once the maintainer situation is a bit [clearer](https://mail.haskell.org/pipermail/libraries/2019-December/030136.html) we can consider removing the benchmark or using a fork of `blaze-builder-0.3` or doing something else…